FGEGraphics/UISystem/UIClickableElement.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using FGEGraphics.UISystem;
+
+namespace FGEGraphics.UISystem;
+
+/// <summary>Represents a clickable UI element on the screen.</summary>
+public abstract class UIClickableElement : UIElement
+{
+    /// <summary>Ran when this element is clicked.</summary>
+    public event EventHandler Clicked;
```

Comment:
Don't use these - just use an `Action` (performance/simplicity/hackability reasons, legacy net EventHandler has a lot of awkward baggage)